README.md

```diff
@@ -108,6 +112,38 @@ This will serve the application with the Gunicorn layer in front of the applicat
 
 This is a simple example of how to test the service after serving on localhost:5000 (`out.dat` will either contain a PNG image or contain an error message.)
 
+## Docker
+
+The service is encapsulated in a Docker image. Images are pushed on the public [Dockerhub](https://hub.docker.com/r/swisstopo/service-color/tags) registry. From each github PR that is merged into develop branch, one Docker image is built and pushed with the following tags:
+
+- `develop.latest`
+- `develop.CURRENT_VERSION-beta.INCREMENTAL_NUMBER`
```

Comment:
Same here it is not `develop.CURRENT_VERSION-beta.INCREMENTAL_NUMBER` but `CURRENT_VERSION-beta.INCREMENTAL_NUMBER`. The branch name is only in the latest tag.

Reply:
:+1: